scalding-core/src/main/scala/com/twitter/scalding/Execution.scala

```diff
@@ -199,6 +199,24 @@ sealed trait Execution[+T] extends java.io.Serializable { self: Product =>
   def zip[U](that: Execution[U]): Execution[(T, U)] =
     Zipped(this, that)
 
+  /**
+   * Add an extra config for current execution.
+   * This changes of config are isolated and affect only current execution.
+   * All changes will disappear for any next operation on execution.
+   */
+  def addConfig(extraConfig: Config): Execution[T] =
```

Comment:
@isnotinvain no, a,b,c will pass to `myExecution`.

but in case:
```
myExecuiton
  .addConfig(a)
  .addConfig(b)
  .addConfig(c)
  .flatMap(item => doSomething(item))
````
`doSomething` don't get a,b,c.